### PR TITLE
Lapin Updates

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary

SUMMARY: Content "Lapin Updates"

#### Purpose of change

Adds additional Mr Lapin content.

#### Describe the solution

Adds a unique faction that gives him the ability to pay for repeatable resource dumps with faction currency of Fur pelts.

#### Describe alternatives you've considered

A dark and twisting questline as Mr Lapin succumbs to the madness inherent in mutation and slowly becomes as evil as the PC probably is.

#### Testing

TBD

### CODE REVIEW

File: data/json/npcs/holdouts/Mr_Lapin.json [hunk 1]
```diff
@@ -134,15 +170,15 @@
     "id": "MISSION_Warrener_FOOD_2",
     "type": "mission_definition",
     "name": { "str": "Sweets" },
-    "description": "Find 25 blackberries.",
```
Comment (Maleclypse) [inline data/json/npcs/holdouts/Mr_Lapin.json:137]: ### ``"description": "Find 25 blackberries.",

File: data/json/npcs/factions.json [hunk 2]
```diff
@@ -579,5 +579,84 @@
...
+        "share my stuff": false,
+        "guard your stuff": false,
+        "lets you in": false,
+        "defends your space": false,
+        "knows your voice": false
+      },
+      "tacoma_commune": {
+        "kill on sight": false,
+        "watch your back": true,
+        "share my stuff": true,
```
Comment (mlangsdorf) [inline data/json/npcs/factions.json:617]: He's that much of an ally with the Tacoma Commune that they'll let him use their stuff without complaint?  Why?

File: data/json/npcs/factions.json [hunk 3]
```diff
@@ -579,5 +579,84 @@
...
+        "watch your back": false,
+        "share my stuff": false,
+        "guard your stuff": false,
+        "lets you in": true,
+        "defends your space": false,
+        "knows your voice": true
+      },
+      "wasteland_scavengers": {
+        "kill on sight": false,
+        "watch your back": true,
```
Comment (mlangsdorf) [inline data/json/npcs/factions.json:641]: Random strangers in the wasteland will defend this guy?

**Review by mlangsdorf — COMMENTED** [review]
I have some questions about the faction relationships.

File: data/json/npcs/factions.json (hunk 3 above)
Comment (Maleclypse) [reply data/json/npcs/factions.json:641]: Huh, I have completely misunderstood how that works.  I thought it was asking if he would defend random strangers and tacoma commune.  I will have to remember this for the future.  Thanks!

**Pull request merged**